A/Agama/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+using BinaryBuilder, Pkg
+
+name = "Agama"
+repo = "https://github.com/GalacticDynamics-Oxford/Agama.git"
+
+version = v"1.0"
```

Comment:
Where is this version coming from? The repository in question doesn't seem to carry any version. 

Food for thought: perhaps it would be better to start out with something like 0.1, in case they start using a version?

Also, I wonder if you have asked them if they would be willing to set a version?